web/html/src/manager/recurring/recurring-playbook-picker.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+
+import { AnsiblePathContent } from "manager/minion/ansible/ansible-path-content";
+
+import { Button } from "components/buttons";
+import { BootstrapPanel } from "components/panels";
+import { SectionToolbar } from "components/section-toolbar/section-toolbar";
+
+type PropsType = {
+  minionServerId: any;
```

Comment:
Here and elsewhere, if you happen to know the types, then it's easier to add them now rather than later.